### ISSUE TITLE
DS215j support

### ISSUE THREAD

**Issue by ngthwi** [cowboy/synology-update-plex#11]
Thanks a lot for your work!

It doesn’t work with DS215j and DS216play, which are armv7hf_neon, because their hardware versions are “DS215jv10-j” and “DS216play-j”. 

[edit] With these 2 NAS, it falls into the case “armv7hf” instead of “armv7hf_neon”

**Comment by cowboy** [comment]
Ok, I'll take a look, thanks!

**Comment by cowboy** [comment]
Can you tell me if my changes in https://github.com/cowboy/synology-update-plex/pull/14 fix the issue?

```sh
$ ssh you@IP_OF_YOUR_NAS
you@yournas:~$ wget "https://raw.githubusercontent.com/cowboy/synology-update-plex/fix-arch/update-plex.sh"
you@yournas:~$ chmod a+x update-plex.sh
```

**Comment by ngthwi** [comment]
There’s a misunderstanding, DS215j and DS216play are armv7hf_neon not armv7hf
(so this fix won’t work)

Best regards

**Comment by cowboy** [comment]
Are you sure `DS216play-j` didn't work correctly? Because I didn't have to change any code, it already returned `armv7hf_neon` for that. Either way, can you check https://github.com/cowboy/synology-update-plex/pull/14 again?

```sh
$ ssh you@IP_OF_YOUR_NAS
you@yournas:~$ wget "https://raw.githubusercontent.com/cowboy/synology-update-plex/fix-arch/update-plex.sh"
you@yournas:~$ chmod a+x update-plex.sh
```

**Comment by ngthwi** [comment]
Hopefully Plex has just released a new version so I could give #14 a try..

So just tested #14 with both DS215j and DS216play and it works fine!
You’re right, DS216play didn’t need to be added to the map. 

Thank you!

**Comment by cowboy** [comment]
v1.0.2 has been released with this fix:

https://github.com/cowboy/synology-update-plex/releases/tag/v1.0.2
